spec/models/mail_alias_spec.rb

```diff
@@ -135,6 +135,25 @@
     end
   end
 
+  describe '#mail_addresses_str' do
+    let(:mail_alias) { FactoryBot.create(:mail_alias, :with_group) }
+    let(:users) { FactoryBot.create_list(:user, 2, groups: [mail_alias.group]) }
+
+    before { users }
+
+    it do
+      expect(["#{users.first.email},#{users.second.email}",
```

Comment:
You could use the `permutation` function here, but this works too.

Reply:
That gives the permutation of the array itself, then you still have to parse it to a string, so probably won't be more clear